ui-guidelines/css/resources-icons.css

```diff
@@ -0,0 +1,609 @@
+}
+
+.ez-icon-home-page:before {
+    content: "\e93e";
+}
+
+.ez-icon-image:before {
+    content: "\e93f";
+}
+
+.ez-icon-information:before {
+    content: "\e940";
+}
+
+.ez-icon-instagram:before {
+    content: "\e941";
+}
+
+.ez-icon-keywords:before {
+    content: "\e942";
+}
+
+.ez-icon-landingpage:before {
+    content: "\e943";
+}
+
+.ez-icon-languages:before {
+    content: "\e944";
+}
+
+.ez-icon-link-remove:before {
+    content: "\e945";
+}
+
+.ez-icon-link:before {
+    content: "\e946";
+}
+
+.ez-icon-lock:before {
+    content: "\e947";
+}
+
+.ez-icon-maform:before {
+    content: "\e948";
+}
+
+.ez-icon-media:before {
+    content: "\e949";
+}
+
+.ez-icon-menu:before {
+    content: "\e94a";
+}
+
+.ez-icon-merge:before {
+    content: "\e94b";
+}
+
+.ez-icon-move:before {
+    content: "\e94c";
+}
+
+.ez-icon-new-tab:before {
+    content: "\e94d";
+}
+
+.ez-icon-news:before {
+    content: "\e94e";
+}
+
+.ez-icon-notice:before {
+    content: "\e94f";
+}
+
+.ez-icon-options:before {
+    content: "\e950";
+}
+
+.ez-icon-pdf-file:before {
+    content: "\e951";
+}
+
+.ez-icon-personalize:before {
+    content: "\e952";
+}
+
+.ez-icon-place-list:before {
+    content: "\e953";
+}
+
+.ez-icon-place:before {
+    content: "\e954";
+}
+
+.ez-icon-portfolio:before {
+    content: "\e955";
+}
+
+.ez-icon-product-dairy:before {
+    content: "\e956";
+}
+
+.ez-icon-product-egg:before {
+    content: "\e957";
+}
+
+.ez-icon-product-gluten:before {
+    content: "\e958";
+}
+
+.ez-icon-product-list:before {
+    content: "\e959";
+}
+
+.ez-icon-product-nuts:before {
+    content: "\e95a";
+}
+
+.ez-icon-product-seafood:before {
+    content: "\e95b";
+}
+
+.ez-icon-product-servings:before {
+    content: "\e95c";
+}
+
+.ez-icon-product-spicy:before {
+    content: "\e95d";
+}
+
+.ez-icon-product-vegetarian:before {
+    content: "\e95e";
+}
+
+.ez-icon-product:before {
+    content: "\e95f";
+}
+
+.ez-icon-publish-later-cancel:before {
+    content: "\e960";
+}
+
+.ez-icon-publish-later:before {
+    content: "\e961";
+}
+
+.ez-icon-publish:before {
+    content: "\e962";
+}
+
+.ez-icon-recommended:before {
+    content: "\e963";
+}
+
+.ez-icon-refresh:before {
+    content: "\e964";
+}
+
+.ez-icon-relations:before {
+    content: "\e965";
+}
+
+.ez-icon-review:before {
+    content: "\e966";
+}
+
+.ez-icon-roles:before {
+    content: "\e967";
+}
+
+.ez-icon-rss:before {
+    content: "\e968";
+}
+
+.ez-icon-save:before {
+    content: "\e969";
+}
+
+.ez-icon-schedule:before {
+    content: "\e96a";
+}
+
+.ez-icon-search:before {
+    content: "\e96b";
+}
+
+.ez-icon-sections:before {
+    content: "\e96c";
+}
+
+.ez-icon-settings-block:before {
+    content: "\e96d";
+}
+
+.ez-icon-shuffle:before {
+    content: "\e96e";
+}
+
+.ez-icon-stats:before {
+    content: "\e96f";
+}
+
+.ez-icon-subscriber:before {
+    content: "\e970";
+}
+
+.ez-icon-swap:before {
+    content: "\e971";
+}
+
+.ez-icon-synonym-add:before {
```

Comment:
I think this is related to eZ Tags.
If so, I don't think we need it in this file